scalableminds/webknossos
# add button to reload data buckets for each layer

This PR adds a button to reload all cached buckets of a data layer in the frontend. 

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a tracing and the dataset settings.
- Click on the reload icon above the histogram of any layer.
- A toast should indicate the success of dropping the cached buckets.
- Now move a bit around to trigger the reloading of the bucket.
- If there are multiple layers, only the data buckets of the selected layer should be reloaded.

### Issues:
- fixes #4335

------
